diff --git a/ClimaCoupler.jl/config/atmos_configs/climaatmos_progedmf_1m.yml b/ClimaCoupler.jl-mod/config/atmos_configs/climaatmos_progedmf_1m.yml
index 3cee8fb..294c3da 100644
--- a/ClimaCoupler.jl/config/atmos_configs/climaatmos_progedmf_1m.yml
+++ b/ClimaCoupler.jl-mod/config/atmos_configs/climaatmos_progedmf_1m.yml
@@ -24,7 +24,7 @@ netcdf_output_at_levels: true
 ode_algo: ARS222
 prescribed_aerosols: ["CB1", "CB2", "DST01", "DST02", "DST03", "DST04", "DST05", "OC1", "OC2", "SO4", "SSLT01", "SSLT02", "SSLT03", "SSLT04", "SSLT05"]
 prognostic_tke: true
-rad: "allskywithclear"
+rad: "allsky"
 rayleigh_sponge: true
 surface_setup: "DefaultMoninObukhov"
 t_end: "120days"
diff --git a/ClimaCoupler.jl/experiments/AMIP/Manifest-v1.11.toml b/ClimaCoupler.jl-mod/experiments/AMIP/Manifest-v1.11.toml
index 2e5b24f..22e9771 100644
--- a/ClimaCoupler.jl/experiments/AMIP/Manifest-v1.11.toml
+++ b/ClimaCoupler.jl-mod/experiments/AMIP/Manifest-v1.11.toml
@@ -404,7 +404,7 @@ weakdeps = ["GeoMakie", "Makie"]
 
 [[deps.ClimaAtmos]]
 deps = ["Adapt", "ArgParse", "Artifacts", "AtmosphericProfilesLibrary", "ClimaComms", "ClimaCore", "ClimaDiagnostics", "ClimaInterpolations", "ClimaParams", "ClimaTimeSteppers", "ClimaUtilities", "CloudMicrophysics", "Dates", "Flux", "ForwardDiff", "Insolation", "Interpolations", "JLD2", "LazyArtifacts", "LazyBroadcast", "LinearAlgebra", "Logging", "NCDatasets", "NVTX", "NullBroadcasts", "RRTMGP", "Random", "SparseMatrixColorings", "StaticArrays", "Statistics", "SurfaceFluxes", "Thermodynamics", "UnrolledUtilities", "YAML"]
-path = "../../../ClimaAtmos.jl"
+path = "../../../ClimaAtmos.jl-mod"
 uuid = "b2c96348-7fb7-4fe0-8da9-78d88439e717"
 version = "0.42.11"
 
@@ -438,7 +438,7 @@ weakdeps = ["CUDA", "MPI"]
 
 [[deps.ClimaCore]]
 deps = ["Adapt", "BandedMatrices", "BlockArrays", "ClimaComms", "ClimaInterpolations", "CubedSphere", "DataStructures", "ForwardDiff", "GaussQuadrature", "GilbertCurves", "HDF5", "InteractiveUtils", "IntervalSets", "KrylovKit", "LLVM", "LazyBroadcast", "LinearAlgebra", "MultiBroadcastFusion", "NVTX", "PkgVersion", "Random", "RootSolvers", "SparseArrays", "StaticArrays", "Statistics", "UnrolledUtilities"]
-path = "../../../ClimaCore.jl"
+path = "../../../ClimaCore.jl-mod"
 uuid = "d414da3d-4745-48bb-8d80-42e94e092884"
 version = "1.0.0"
 weakdeps = ["CUDA", "GPUCompiler", "Krylov", "Makie", "RecipesBase"]
@@ -2473,7 +2473,7 @@ version = "1.11.0"
 
 [[deps.RRTMGP]]
 deps = ["Adapt", "Artifacts", "ClimaComms", "PrecompileTools", "Random", "Serialization"]
-path = "../../../RRTMGP.jl"
+path = "../../../RRTMGP.jl-mod"
 uuid = "a01a1ee8-cea4-48fc-987c-fc7878d79da1"
 version = "1.0.0"
 weakdeps = ["CUDA", "ClimaParams", "NCDatasets"]
